CHANGELOG.rst

```diff
@@ -11,6 +11,7 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 =============
 Fixed
 -----
+Corrected ATT error code 15 description to Insufficient Encryption
```

Comment:
It looks like somehow we ended up with duplicate Fixes headings. There is another one below that already has entries, so we should move this with the rest and delete this header.